### PR TITLE
Fix: Fixing line height 

### PR DESCRIPTION
**Pull request by Delyc**
## Describe your changes
Fixing line height on who we are section

## Issue ticket number and link
- ID: 
- Link: 

## Tasks completed
- [x] Fixing line height on who we are section

 
## Tasks not completed
- None

### CODE REVIEW

**Review by alain-kubwayo — APPROVED** [review]
Looks good!

File: tailwind.config.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ module.exports = {
         7.15:"1.7875rem",
         7.5:"1.875rem",
         7.83: "1.958rem",
+        9: "2.25rem",
```
Comment (Ntwali-Josue) [inline tailwind.config.js:64]: By default, we do have this line-height value in Tailwind, so you don't need to add it here.

**Review by Ntwali-Josue — CHANGES_REQUESTED** [review]
Hey @Delyc, good work on fixing the line height and colors of the `What We Do` section. There is one requested change inside the `tailwind.config.js`, work on it for your PR to be ready for approval.

**Review by mhenriette — APPROVED** [review]
Looks Good!!

**Review by Ntwali-Josue — APPROVED** [review]

**Pull request merged**